instrumentation/servlet/servlet-3.0/javaagent/src/test/groovy/TomcatServlet3Test.groovy

```diff
@@ -120,6 +123,53 @@ abstract class TomcatServlet3Test extends AbstractServlet3Test<Tomcat, Context>
     Tomcat.addServlet(servletContext, name, servlet.newInstance())
     servletContext.addServletMappingDecoded(path, name)
   }
+//
+//  def "snippet injection"() {
+//    setup:
```

Comment:
if uncomment this, and run "snippet injection" twice, the second test would fail